cli-commands/nodeos/subcommands/show/subcommands/logs/index.js

```diff
@@ -0,0 +1,34 @@
+const exec = require('child_process').exec;
+const Command = require('../../../../../command');
+
+const commandMessages = require('./messages');
+const showCommandDefinition = require('./definition');
+
+// eoslime nodeos show logs --path --lines
+
+class LogsCommand extends Command {
+    constructor() {
+        super(showCommandDefinition);
+    }
+
+    async execute(args) {
+        try {
+            const optionsResults = await super.processOptions(args);
+            await showLogs(optionsResults);
+        } catch (error) {
+            commandMessages.UnsuccessfulShowing(error);
+        }
+    }
+}
+
+const showLogs = async function (result) {
+    exec(`tail -n ${result.lines} ${result.path}`, (error, stdout, stderr) => {
```

Comment:
Updated to use AsyncSoftExec. Implementation of AsyncSoftExec updated as well in order to handle the result on success, since we need in that case the log lines in order to be displayed.